bin/apply-core-fork-patch.sh

```diff
@@ -37,7 +38,7 @@ applyPatch () {
 #   None
 #######################################
 createPatch () {
-  curl "$API_URL_BASE/compare/$1...$2" -s -H "Accept: application/vnd.github.v3.diff" > "$civiRoot/$PATCH_FILE"
+  curl "$REPO_BASE_URL/compare/$1...$2.diff" -s -H "Accept: application/vnd.github.v3.diff" > "$civiRoot/$PATCH_FILE"
```

Comment:
That we're not using the API anymore, that `-H` option isn't necessary

Reply:
It makes sense since any update in API (to version 4 or so) would have been catered for by URL and so will not affect the script. 👍